forge-gui/res/cardsfolder/upcoming/genku_future_shaper.txt

```diff
@@ -0,0 +1,12 @@
+Name:Genku, Future Shaper
+ManaCost:2 W U
+Types:Legendary Creature Moonfolk Wizard
+PT:2/5
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Permanent.YouCtrl+nonToken | TriggerZones$ Battlefield | Execute$ TrigCharm | TriggerDescription$ Whenever another nontoken permanent you control leaves the battlefield, choose one that hasn't been chosen this turn. Create a creature token with those characteristics.
```

Comment:
description incomplete?

Reply:
This is the same description format as Outlaws' Merriment I think. The token descriptions appear on the `Token` lines

Reply:
It seems incomplete
When in doubt it's always good to confirm ingame if everything shows up:
![grafik](https://github.com/Card-Forge/forge/assets/8506892/aee02747-7673-4c21-bc34-38f1b3a3507e)

Reply:
I see the full list of tokens isn't being included in the card's hover-over text. I can't for the life of me figure out why this is the case when Outlaws' Merriment appears to not have this problem, but is scripted the same way? (I.e. the list of tokens isn't hard-baked into the trigger description.)

If I copy-paste the `\n• 2/2 white Fox with vigilance.\n• 1/2 blue Moonfolk with flying.\n• 1/1 black Rat with lifelink.` text into the trigger description, the hover-over text reflects this and looks fine, but the trigger description on the stack itself starts to look very ugly (the newline esc characters appear in the raw text). Is this only solvable with a unique `StackDescription` or am I approaching this wrong?

Reply:
you need to use ABILITY syntax to tell the engine to include the charm description

Reply:
Will fix after merge